src/webview_provider/queryResultPanel.ts

```diff
@@ -332,20 +332,16 @@ export class QueryResultPanel implements WebviewViewProvider {
   }
 
   /** Runs a query transmitting appropriate notifications to webview */
-  public async executeQuery(
-    query: string,
-    queryExecutionPromise: Promise<QueryExecution>,
-  ) {
+  public async executeQuery(query: string, queryExecution: QueryExecution) {
     //using id to focus on the webview is more reliable than using the view title
     await commands.executeCommand("dbtPowerUser.PreviewResults.focus");
     if (this._panel) {
       this._panel.show(); // Show the view
       this._panel.webview.postMessage({ command: "focus" }); // keyboard focus
       this.transmitLoading();
     }
+    this.queryExecution = queryExecution;
```

Comment:
is this intentional? @suryaiyer95 made this change: https://github.com/AltimateAI/vscode-dbt-power-user/pull/925/files